config-builds

```diff
@@ -23,6 +23,7 @@ CC=clang CXX=clang++ $CMAKE -B build.tmp/clang-release \
     -DCMAKE_BUILD_TYPE=Release
 
 
+# We can't use `undefined` sanitizer with coroutines due to clang bug :-(
```

Comment:
If you have any kind of reference to the exact bug, you may thank yourself later for including it here.

Reply:
I'm not 100% sure that it is a clang bug, I just know that the undefined sanitizer doesn't seem to work with coroutines.